asmcomp/asmlink.ml

```diff
@@ -555,10 +555,17 @@ let call_linker file_list_rev startup_file output_name =
       else if needs_objcopy_workflow then
         (* Run objcopy to extract debug info into .debug file *)
         let debug_file = output_name ^ ".debug" in
+        let compression_flag = 
+          match Dwarf_flags.get_dwarf_compression_format () with
+          | Some compression -> 
+            Printf.sprintf " --compress-debug-sections=%s" compression
+          | None -> ""
+        in
         let objcopy_cmd = 
           Printf.sprintf 
```

Comment:
(Related to the discussion on the other
PR, maybe `--enable-deterministic-archives`
should be passed.)

Reply:
This is a binutils configuration option, no?

Reply:
Ah, I see - in which case it should be passed I think.